libraries/AP_GPS/AP_GPS_config.h

```diff
@@ -104,3 +104,20 @@
 #ifndef HAL_GPS_COM_PORT_DEFAULT
 #define HAL_GPS_COM_PORT_DEFAULT 1
 #endif
+
+
+#ifndef AP_GPS_GPS_RAW_INT_SENDING_ENABLED
```

Comment:
I don't think we want or need to be able to control these individually. Instead a single define AP_GPS_LEGACY_MESSAGE_SENDING_ENABLED would be a lot simpler

Reply:
I've pushed up a patch which uses the new more-details defines to save 300-odd bytes on boards that don't have either of ERB or SBF:

```
Board                    AP_Periph  blimp  bootloader  copter  heli  iofirmware  plane  rover  sub
CubeOrange-periph-heavy  *                                                                     
CubeRedPrimary                      *      *           *       *                 *      *      *
Durandal                            *      *           *       *                 *      *      *
Hitec-Airspeed           *                                                                     
KakuteH7-bdshot                     *      *           *       *                 *      *      *
MatekF405                           -312   *           -312    -312              -312   -320   -312
Pixhawk1-1M-bdshot                  -312               -312    -312              -312   -304   -312
f103-QiotekPeriph        *                                                                     
f303-Universal           *                                                                     
iomcu                                                                *                         
revo-mini                           -312   *           -312    -312              -312   -312   -312
skyviper-journey                                       *                                       
skyviper-v2450                                         -280                                    
```

Reply:
.... also, if you only have a single GPS (https://github.com/ArduPilot/ardupilot/pull/26751) we can save a few hundred more bytes for a year or two.